lib/matplotlib/axes/_axes.py

```diff
@@ -1757,13 +1757,26 @@ def plot(self, *args, scalex=True, scaley=True, data=None, **kwargs):
         self._request_autoscale_view(scalex=scalex, scaley=scaley)
         return lines
 
+    @cbook.deprecated("3.4", pending=True,
+                      alternative="plot() and if needed Axis.axis_date()")
     @_preprocess_data(replace_names=["x", "y"], label_namer="y")
     @docstring.dedent_interpd
     def plot_date(self, x, y, fmt='o', tz=None, xdate=True, ydate=False,
                   **kwargs):
         """
         Plot data that contains dates.
 
+        .. note::
+
+            This method exists for historic reasons and will be deprecated in
+            the future.
+
+            - ``datetime`` data should directly be plotted using `~.Axes.plot`.
```

Comment:
is ``datetime`` good enough, or should we also specify `np.datetime64`?

Reply:
Maybe ``datetime``-like?